internal/teststep/verify_import_plan_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package teststep_test
+
+import (
+	"testing"
+
+	tfjson "github.com/hashicorp/terraform-json"
+	"github.com/hashicorp/terraform-plugin-testing/internal/teststep"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+)
+
+func Test_VerifyImportPlan(t *testing.T) {
```

Comment:
One nice thing for this test would be to expand it for the other types of data outside of primitives (lists, sets, maps, single nested object/block, list nested objects/blocks, set nested objects/blocks, map nested attribute)